.github/workflows/test.sh

```diff
@@ -13,7 +13,26 @@ prun cd example_pkg
 prun spin --version
 
 pip install meson-python ninja
-prun spin build
+
+
+# Test spin build + debug builds
+echo -e "${MAGENTA}Creating debug builds${NORMAL}"
```

Comment:
Please see the `ptest` command that has been introduced on `main`

Reply:
Refined in [02e9e63](https://github.com/scientific-python/spin/pull/146/commits/02e9e6356ec55708210fa8702b410adc21a9ed02)